
# HG changeset patch
# User Ryan C. Gordon <icculus@icculus.org>
# Date 1269547127 14400
# Node ID ee3d2e6e1161799e9b6822eb7ec589a4dea9a352
# Parent  01e7571471efc54c036a558360518c5088c87c11
Initialize an array to quiet compiler complaining.

Some versions of GCC think this can be used uninitialized. It doesn't appear
 to be the case, but we initialize it to keep the compiler happy. It's probably
 a good idea anyhow.

--- a/archivers/zip.c	Wed Mar 24 23:51:40 2010 -0400
+++ b/archivers/zip.c	Thu Mar 25 15:58:47 2010 -0400
@@ -375,7 +375,7 @@
 static PHYSFS_sint64 zip_find_end_of_central_dir(void *in, PHYSFS_sint64 *len)
 {
     PHYSFS_uint8 buf[256];
-    PHYSFS_uint8 extra[4];
+    PHYSFS_uint8 extra[4] = { 0, 0, 0, 0 };
     PHYSFS_sint32 i = 0;
     PHYSFS_sint64 filelen;
     PHYSFS_sint64 filepos;

